src/react-app-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```

Comment:
What's this file for? Should it be in gitignore?

Reply:
This is a TypeScript declaration file, kind of like a header file in C/C++. It should be committed and is also present in the API wrapper PR.